sale_commission/security/ir.model.access.csv

```diff
@@ -1,5 +1,6 @@
 "id","name","model_id:id","group_id:id","perm_read","perm_write","perm_create","perm_unlink"
 "access_sale_commission_manager","access_sale_commission_manager","model_sale_commission","base.group_sale_manager",1,1,1,1
+"access_res_partner_agent","access_res_partner_agent","model_res_partner_agent","base.group_sale_manager",1,1,1,1
```

Comment:
Needs minimum READ access in base.group_user for agent_lines_company as it is added to res.partner using One2many field